hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -4216,6 +4216,20 @@
     </description>
   </property>
 
+  <property>
```

Comment:
Should we have a similar config for other components like scm as well?

Reply:
Thanks @swamirishi  for the review. Yes we could do that, I see DatanodeConfiguration already has this [defined ](https://github.com/apache/ozone/blob/404a036348ef1cbddf24db8db914e5149f03d886/hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/DatanodeConfiguration.java#L509)but it is not used while instantiating the DB. We could also add this for SCM.  Will open a Jira.